src/www/services_dhcpv6.php

```diff
@@ -430,8 +430,10 @@ function show_netboot_config() {
                     <tr>
                       <td><i class="fa fa-info-circle text-muted"></i>  <?=gettext("Enable");?></td>
                       <td>
-                        <input name="enable" type="checkbox" value="yes" <?php if ($pconfig['enable']) echo "checked=\"checked\""; ?> />
-                        <strong><?= sprintf(gettext("Enable DHCPv6 server on " . "%s " ."interface"),!empty($config['interfaces'][$if]['descr']) ? htmlspecialchars($config['interfaces'][$if]['descr']) : strtoupper($if));?></strong>
+                        <label for="enable">
+                          <input name="enable" id="enable" type="checkbox" value="yes" <?php if ($pconfig['enable']) echo "checked=\"checked\""; ?> />
+                          <strong><?= sprintf(gettext("Enable DHCPv6 server on " . "%s " ."interface"),!empty($config['interfaces'][$if]['descr']) ? htmlspecialchars($config['interfaces'][$if]['descr']) : strtoupper($if));?></strong>
```

Comment:
please fix the string

Reply:
If you wish to change the string you can create a commit.  Merge conflicts will then be resolved in this commit.

Reply:
it is just about a removal of this useless string concatenation - the gettext string does not need that.

Reply:
Submit a commit to change it.  That is in the master.  Not something introduced here.